api/v1alpha1/cluster_types.go

```diff
@@ -28,94 +28,72 @@ import (
 	// +optional
 	NodeTLSSecret string `json:"nodeTLSSecret,omitempty"`
 	// (Optional) The secret with a certificate and a private key for root database user
 	// Default: ""
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Client TLS Secret Name",xDescriptors="urn:alm:descriptor:io.kubernetes:Secret"
 	// +optional
 	ClientTLSSecret string `json:"clientTLSSecret,omitempty"`
-	// The maximum number of pods that can be unavailable during a rolling update.
+	// (Optional) The maximum number of pods that can be unavailable during a rolling update.
 	// This number is set in the PodDistruptionBudget and defaults to 1.
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="PDB Max Unavailable Nodes",xDescriptors="urn:alm:descriptor:com.tectonic.ui:number"
 	// +optional
 	MaxUnavailable *int32 `json:"maxUnavailable,omitempty"`
-	// The min number of pods that can be unavailable during a rolling update.
+	// (Optional) The min number of pods that can be unavailable during a rolling update.
 	// This number is set in the PodDistruptionBudget and defaults to 1.
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="PDB Min Available Nodes",xDescriptors="urn:alm:descriptor:com.tectonic.ui:number"
 	// +optional
 	MinAvailable *int32 `json:"minUnavailable,omitempty"`
 	// (Optional) The total size for caches (`--cache` command line parameter)
 	// Default: "25%"
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Cache Size",xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"
 	// +optional
 	Cache string `json:"cache,omitempty"`
 	// (Optional) The maximum in-memory storage capacity available to store temporary
 	// data for SQL queries (`--max-sql-memory` parameter)
 	// Default: "25%"
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Max SQL Memory",xDescriptors="urn:alm:descriptor:com.tectonic.ui:text"
 	// +optional
 	MaxSQLMemory string `json:"maxSQLMemory,omitempty"`
 	// (Optional) Additional command line arguments for the `cockroach` binary
 	// Default: ""
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="List of additional arguments"
 	// +optional
 	AdditionalArgs []string `json:"additionalArgs,omitempty"`
 	// (Optional) Database container resource limits. Any container limits
 	// can be specified.
 	// Default: (not specified)
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Pod Resource Requirements"
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 	// (Required) Database disk storage configuration
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Storage Volume Configuration"
 	// +required
 	DataStore Volume `json:"dataStore,omitempty"`
 }
 
 // CrdbClusterStatus defines the observed state of Cluster
 type CrdbClusterStatus struct {
-	// Important: Run "make" to regenerate code after modifying this file
```

Comment:
This seems like an import comment to keep around? Should this just be at the top of the file?